src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -598,6 +618,269 @@ public static unsafe int IndexOfAny(ref byte searchSpace, byte value0, byte valu
             return (int)(byte*)(index + 7);
         }
 
+        public static unsafe int LastIndexOfAny(ref byte searchSpace, byte value0, byte value1, int length)
+        {
+            Debug.Assert(length >= 0);
+
+            uint uValue0 = value0; // Use uint for comparisons to avoid unnecessary 8->32 extensions
+            uint uValue1 = value1; // Use uint for comparisons to avoid unnecessary 8->32 extensions
+            IntPtr index = (IntPtr)(uint)length; // Use UIntPtr for arithmetic to avoid unnecessary 64->32->64 truncations
+            IntPtr nLength = (IntPtr)(uint)length;
+#if !netstandard11
+            if (Vector.IsHardwareAccelerated && length >= Vector<byte>.Count * 2)
```

Comment:
Some more comments in this code would be useful.

Reply:
I won't block on this (since CI is green), but will keep a note of this, especially if we were to re-work some of the implementation details.